examples/httpserver/pubspec.yaml

```diff
@@ -3,10 +3,11 @@ description: Sample HTTP clients and servers
 homepage: https://www.dartlang.org/docs/tutorials/httpserver/
 
 environment:
-  sdk: '>=2.1.0 <3.0.0'
+  sdk: '>=2.3.1 <3.0.0'
```

Comment:
I think travis is unhappy with this change:

```
Because httpserver requires SDK version >=2.3.1 <3.0.0, version solving failed.
```

Can we set it to 2.3.0 or do we need to upgrade the SDK?

Reply:
I think we should use 2.3.1, since that's the latest stable version. Let me see what I can do to make that stick...

Reply:
Turns out I filed a bug to remind myself to do this (or something similar, anyway): #1469.